### DISCUSSION TITLE
How to enable DAG parsed time in DAG view?

### DISCUSSION THREAD

**Discussion by andresitorusz** [apache/airflow#33163] [Q&A]
Hi everyone,

I have a pre-deployed Airflow in a Kubernetes cluster, how can I enable the DAG parsed time from this [PR](https://github.com/apache/airflow/pull/27573) without redeploying the Airflow?

Thanks in advance.

**Comment by potiuk** [comment]
If you want to shoot yourself in the foot, you can try to attempt to cherry-pick that PR to whatever version of Airflow you are running. You are opening to all kinds of issues and problems when you do it, and you will be left completely alone with your own modified version of airflow that might or might not work, and nobody will be able to help you with solving the problems.

Or you might upgrade to (ideally) latest version of Airflow to get it.

The choice is yours.
